Document/0x06g-Testing-Network-Communication.md

```diff
@@ -197,6 +197,14 @@ In order to address this risk you can use certificate pinning. Certificate pinni
 The certificate can be pinned during development, or at the time the app first connects to the backend.
 In that case, the certificate associated or 'pinned' to the host at when it seen for the first time. This second variant is slightly less secure, as an attacker intercepting the initial connection could inject their own certificate.
 
+Note that there are various ways to work with a failing pin:
```

Comment:
the points below are only partially in sync with the points in the other chapter, is this intended? or should they be completely in sync?

Another question: If they are the same points. Shouldn't be added to 0x04f? I'm feeling like there's a lot being repeated here. Maybe all this generic info could be added there and referenced from here. That would leave this 2 chapters with:
- a little intro that will reference the generic section in 0x04f.
- only the Android/iOS related part (what's in static and dynamic analysis).